mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/MainActivity.java

```diff
@@ -87,18 +95,45 @@
   // Contains dynamic layout of sticker data controller
   private LinearLayout buttonLayout;
 
-  private ArrayList<Sticker> stickerList;
+  private ArrayList<Sticker> stickerArrayList;
   private Sticker currentSticker; // Sticker being edited
 
+  // Define parameters for 'reactivity' of object
+  private final float ROTATION_SPEED = 5.0f;
+  private final float SCALING_FACTOR = 0.025f;
+
+  // Parameters of device visual field for rendering system
+  // (68 degrees, 4:3 for Pixel 4)
+  // TODO: Make acquisition of this information automated
+  private final float VERTICAL_FOV_RADIANS = (float)Math.toRadians(68.0);
+  private final float ASPECT_RATIO = (4.0f/3.0f);
+  private final String FOV_SIDE_PACKET_TAG = "vertical_fov_radians";
+  private final String ASPECT_RATIO_SIDE_PACKET_TAG = "aspect_ratio";
+
   private float[] imuData = new float[3]; // [roll,pitch,yaw]
 
+  private static final String STICKER_PROTO_TAG = "sticker_proto_string";
+  private static final String IMU_DATA_TAG = "imu_data";
   // Assets for object rendering
-  private Bitmap objTexture = null;
-  private static final String OBJ_TEXTURE = "chair_texture.bmp";
-  private static final String OBJ_FILE = "chair.obj.uuu";
-  // Tags for the side packets of model texture and .obj.uuu
-  private static final String SIDE_PACKET_ASSET_TAG = "obj_asset_name";
-  private static final String SIDE_PACKET_TEXTURE_TAG = "obj_texture";
+  // All robot animation assets and tags
+  private Bitmap robotTexture = null;
+  private static final String ROBOT_TEXTURE = "robot_texture.bmp";
+  private static final String ROBOT_FILE = "robot.obj.uuu";
+  private static final String ROBOT_TEXTURE_TAG = "robot_texture";
+  private static final String ROBOT_ASSET_TAG = "robot_asset_name";
+  // All dino animation assets and tags
+  private Bitmap dinoTexture = null;
+  private static final String DINO_TEXTURE = "dino_texture.bmp";
```

Comment:
Maybe add a TODO to try to use something more compressed like webp or png in the future-- bmp's are, unfortunately, huge.  Not important to address now though.